movie.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'item'
+class Movie < Item
+  attr_accessor :id, :name, :publish_date, :silet
+
+  def initialize(name, publish_date, silet)
+    super(publish_date)
+    @id = Random.rand(1...1000)
+    @name = name
+    @silet = silet
+  end
+
+  def can_be_archived?
+    super && @silet == true
+  end
+end
```

Comment:
The project requirement for this method is to return true if parent's method returns true OR if silent equals true, so you should put || logical operator instrad of &&.